docs/data/whats-new.md

```diff
@@ -0,0 +1,20 @@
+# ✨ What's new in v6? ✨
+
+<p class="description">Discover the new features on the latest major version.</p>
+
+## Data Grid
+
+- [ApiRef available in the community version](https://next.mui.com/x/react-data-grid/api-object/)
+- [Improved column menu](https://next.mui.com/x/react-data-grid/column-menu/)
+- [Cell selection range](https://next.mui.com/x/react-data-grid/cell-selection/) [<span class="plan-premium"></span>](/x/introduction/licensing/#pro-premium)
+
+## Date and Time pickers
+
+- [Fields: the new default input for pickers](https://next.mui.com/x/react-date-pickers/fields/).
+- [Improved layout customization](https://next.mui.com/x/react-date-pickers/custom-layout/)
+- [Shortcuts for picking specific dates in a calendar](https://next.mui.com/x/react-date-pickers/shortcuts/)
+- [Edit date ranges with drag and drop](https://next.mui.com/x/react-date-pickers/date-range-calendar/) [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan)
+
+## Migrating from v5?
+
+Migration guides are available for both the [Data Grid](https://next.mui.com/x/migration/migration-data-grid-v5/) and [Date Pickers](https://next.mui.com/x/migration/migration-pickers-v5/), with a complete list of breaking changes and a few [codemods](https://github.com/mui/mui-x/blob/next/packages/x-codemod/README.md) to support your migrating experience.
```

Comment:
I wouldn't say that we have "a few" codemods and it is ever increasing as well. 🤔 🙈 

```suggestion
Migration guides are available for both the [Data Grid](https://next.mui.com/x/migration/migration-data-grid-v5/) and [Date Pickers](https://next.mui.com/x/migration/migration-pickers-v5/), with a complete list of breaking changes and [codemods](https://github.com/mui/mui-x/blob/next/packages/x-codemod/README.md) to support your migrating experience.
```